src/Event.php

```diff
@@ -415,9 +422,136 @@ public static function showList($target, $order = 'DESC', $sort = 'date', $start
+            'additionalfields' => ['type'],
+        ];
+
+        $tab[] = [
+            'id'            => '157',
+            'table'         => self::getTable(),
+            'field'         => 'date',
+            'name'          => __('Date'),
+            'datatype'      => 'datetime',
+            'massiveaction' => false,
+        ];
+
+        $tab[] = [
+            'id'            => '158',
+            'table'         => self::getTable(),
+            'field'         => 'service',
+            'name'          => __('Service'),
+            'datatype'      => 'specific',
+            'massiveaction' => false,
+            'searchtype'    => ['equals', 'notequals', 'contains', 'notcontains'],
+        ];
+
+        $tab[] = [
+            'id'            => '159',
+            'table'         => self::getTable(),
+            'field'         => 'level',
+            'name'          => __('Level'),
+            'datatype'      => 'integer',
+            'massiveaction' => false,
+        ];
+
+        $tab[] = [
+            'id'            => '160',
+            'table'         => self::getTable(),
+            'field'         => 'message',
+            'name'          => __('Message'),
+            'datatype'      => 'text',
+            'massiveaction' => false,
+        ];
+
+        return $tab;
+    }
+
+    public static function getSpecificValueToSelect($field, $name = '', $values = '', array $options = [])
+    {
+        if ($field === 'service') {
+            $value = $values['service'];
+            if (empty($value)) {
+                $value = 0;
+            }
+            return \Dropdown::showFromArray($name, self::logArray()[1], [
+                'value' => $value,
+                'display' => false,
+                'display_emptychoice' => true
+            ]);
+        } else if ($field === 'type') {
+            $value = $values['type'];
+            if (empty($value)) {
+                $value = 0;
+            }
+            return \Dropdown::showFromArray($name, self::logArray()[0], [
+                'value' => $value,
+                'display' => false,
+                'display_emptychoice' => true
+            ]);
+        }
+        return parent::getSpecificValueToSelect($field, $name, $values, $options);
+    }
+
+    public static function getSpecificValueToDisplay($field, $values, array $options = [])
+    {
+        if ($field === 'service') {
+            $value = $values['service'];
+            if (empty($value)) {
+                return NOT_AVAILABLE;
+            }
+            $services = self::logArray()[1];
+            return $services[$value] ?? $value;
+        } else if ($field === 'items_id') {
+            $type = $values['type'] ?? null;
+            if (((int) $values['items_id']) > 0 && $type !== null && is_subclass_of($type, CommonGLPI::class)) {
+                $url = $type::getFormURLWithID($values['items_id']);
+                return Html::link($values['items_id'], $url);
```

Comment:
Not sure we will have any `CommonGLPI` item having an ID, so, by limiting this case to `CommonDBTM` items, we could display their name instead of just an ID.
```suggestion
            if (((int) $values['items_id']) > 0 && $type !== null && is_subclass_of($type, CommonDBTM::class, true)) {
                $item = new $type();
                if ($type->getFromDB($values['items_id'])) {
                    return $type->getLink();
                }
```